mavros/msg/ActuatorControl.msg

```diff
@@ -0,0 +1,12 @@
+# raw servo values for direct actuator controls
+# 0: roll
+# 1: pitch
+# 2: yaw
+# 3: throttle
+# 4: flaps
+# 5: spoilers
+# 6: airbrakes
+# 7: landing gear
+
+Header header
+float32[] channels
```

Comment:
RCOut also has channels[]  (without a size) ? why is that?

Am 01.03.2015 um 13:55 schrieb Vladimir Ermakov:
>
> In mavros/msg/ActuatorControl.msg
> <https://github.com/mavlink/mavros/pull/230#discussion_r25567060>:
>
> > @@ -0,0 +1,12 @@
> > +# raw servo values for direct actuator controls
> > +# 0: roll
> > +# 1: pitch
> > +# 2: yaw
> > +# 3: throttle
> > +# 4: flaps
> > +# 5: spoilers
> > +# 6: airbrakes
> > +# 7: landing gear
> > +
> > +Header header
> > +float32[] channels
>
> Also change [] → [8]. And last thing is find how get pointer from
> |boost::array<float, 8>|.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/mavros/pull/230/files#r25567060>.
>